src/Datasource/QueryInterface.php

```diff
@@ -18,7 +18,7 @@
 /**
  * The basis for every query object
  *
- * @package Cake\Datasource
+ * @method $this andWhere($conditions, $types = [])
```

Comment:
We could change the callers to use `where()` instead of expanding the interface here.